.travis.yml

```diff
@@ -78,6 +83,17 @@ before_install:
    fi
  - ccache -s
  - gem install coveralls-lcov
+ # CMAKE most recent version
+ - >
+    if [ "$(ls -A ${CMAKE_INSTALL})" ]; then
+      echo "CMake found in cache.";
+    else
+      mkdir --parent ${CMAKE_SOURCE}
+      mkdir --parent ${CMAKE_INSTALL}
+      travis_retry wget --no-check-certificate --quiet -O - ${CMAKE_URL} | tar --strip-components=1 -xz -C ${CMAKE_INSTALL}
+    fi
+ - export PATH=${CMAKE_INSTALL}/bin:${PATH};
+ - cmake --version
```

Comment:
at https://github.com/poparteu/openMVG/blob/new_enhanced_logs/.travis.yml#L191 remember to add `log` as module to build for boost